common/views/components/BaseTabs/BaseTabs.tsx

```diff
@@ -46,14 +46,14 @@ const Tab = styled.button.attrs((props: TabProps) => ({
 
 type TabPanelProps = {
   id: string;
-  isActive: boolean;
+  isHidden: boolean;
   isEnhanced: boolean;
 };
 const TabPanel = styled.div.attrs((props: TabPanelProps) => ({
   id: props.id,
   role: props.isEnhanced ? 'tabpanel' : undefined,
-  hidden: props.isEnhanced ? !props.isActive : undefined,
-  'aria-expanded': props.isEnhanced ? props.isActive : undefined,
+  hidden: props.isEnhanced ? props.isHidden : undefined,
```

Comment:
You call - but I meant having an isHidden flag without the isEnhanced flag, keeping the logic closer to the component.

e.g.
```TS
<TabPanel isHidden={props.isEnhanced ? id !== activeId : false} />
```

Reply:
But just realised we use the isEnhanced elasewhere.